oauth2_provider/oauth2_validators.py

```diff
@@ -240,13 +240,25 @@ def validate_client_id(self, client_id, request, *args, **kwargs):
     def get_default_redirect_uri(self, client_id, request, *args, **kwargs):
         return request.client.default_redirect_uri
 
-    def _get_token_from_authentication_server(self, token, introspection_url, introspection_token):
-        bearer = "Bearer {}".format(introspection_token)
+    def _get_token_from_authentication_server(self,
+                                              token,
+                                              introspection_url,
+                                              introspection_token,
+                                              introspection_credentials):
```

Comment:
minor: please use indented style instead of hanging brackets:

```py
def _get_token_from_authentication_server(
    self, token, introspection_url, introspection_token, introspection_credentials
):
    ...
```